packages/apps/human-app/frontend/src/modules/worker/components/profile/profile-email-notifications.tsx

```diff
@@ -1,25 +0,0 @@
-import Grid from '@mui/material/Grid';
-import { useTranslation } from 'react-i18next';
-import Typography from '@mui/material/Typography';
-import Stack from '@mui/material/Stack';
-import Switch from '@mui/material/Switch';
-import { useAuthenticatedUser } from '@/modules/auth/hooks/use-authenticated-user';
-
-export function ProfileEmailNotification() {
```

Comment:
Would be nice to also remove it's commented usage in `WorkerProfilePage` component